docs/supported_models.md

```diff
@@ -143,6 +143,7 @@
 |signify        |LCT016                   |Philips Hue 800 Lumen White and Color A19                                              |                                                                                                                                                                                                                      |
 |signify        |LCT024                   |Hue Play                                                                               |                                                                                                                                                                                                          440400982841|
 |signify        |LCT026                   |Hue Go 2.0                                                                             |7602031P7                                                                                                                                                                                                             |
+|signify        |LCX004                   |Hue Gradient Lightstrip                                                                |                                                                                                                                                                                                           |
```

Comment:
No need to alter this file manually, it is auto generated. No issue for now it will just be overwritten after this PR is merged.